parser.c

```diff
@@ -504,8 +504,6 @@ void process_pihole_log(int file)
 				// Debug output
 				if(strlen(hostname) > 0)
 				{
-					// Convert hostname to lower case
```

Comment:
We don't need to do it here, as it is now guaranteed that `char *resolveHostname(const char *addr)` returns a lower case host name.